sv.c

```diff
@@ -16030,6 +16039,24 @@ Perl_clone_params_del(CLONE_PARAMS *param)
     }
 }
 
+/*
+=for apidoc clone_params_new
+
+Allocate and initialize a C<CLONE_PARAMS> structure for the
+S<C<PerlInterpreter *>> parameter C<to> based on data contained in the C<from>
+S<C<PerlInterpreter *>> parameter, returning a pointer to the new structure.
```

Comment:
I'm not sure this is right. And even if it is, I'm not sure this should be API because it's surprisingly unhelpful.

Reply:
If sv_dup is made API, and this isn't, how to resolve the fact that sv_dup has a parameter that may not be NULL that one of these fills, and the other frees?   I see on cpan some uses that fill that structure manually, which is diving even more into the internals.

Reply:
> If sv_dup is made API, and this isn't, how to resolve the fact that sv_dup has a parameter that may not be NULL that one of these fills, and the other frees? I see on cpan some uses that fill that structure manually, which is diving even more into the internals.

And they're buggy for doing that.

`svt_dup` gets a `CLONE_PARAMS*` argument, but somehow `CLONE` doesn't. Recreating a new one for an existing clone session can easily lead to wrong results. I have a patch for that on my todo list.